### PR TITLE
Add pre-release builds/tests for CentOS Stream 9.

### PR DESCRIPTION
**Pull request by zmarano**
Also, add Debian 11 to our integ tests.

### CODE REVIEW

**Review by hopkiw — CHANGES_REQUESTED** [review]

File: concourse/pipelines/guest-package-build.yaml [hunk 1]
```diff
@@ -167,10 +175,16 @@ jobs:
           source-image: projects/rhel-cloud/global/images/family/rhel-8
           dest-image: rhel-8-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent-((.:package-version))-g1.el8.x86_64.rpm
+      - task: build-package-image-el-9
```
[the remark this answers is not in the document]
Comment (zmarano) [reply concourse/pipelines/guest-package-build.yaml:178]: Removed the test for now. el-9 was consistent with el-8 and el-7.

**Review by hopkiw — APPROVED** [review]

**Review by hopkiw — APPROVED** [review]

**Pull request merged**